### PR TITLE
Setup i18n

### PR DESCRIPTION
**Pull request by tmuntaner**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false"><table>
  <tbody>
    <tr>
      <td>lib/rmt/cli/products.rb</td>
      <td>99</td>
      <td>Layout/SpaceAfterComma: Space missing after comma. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>
</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false"><table>
  <tbody>
    <tr>
      <td>lib/rmt/cli/products.rb</td>
      <td>99</td>
      <td>Metrics/LineLength: Line is too long. [212/160] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)</td>
    </tr>
  </tbody>
</table>
</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: config/boot.rb [hunk 1]
```diff
@@ -5,6 +5,21 @@
...
+require 'fast_gettext'
 
 Rails::Command::ServerCommand.send(:remove_const, 'DEFAULT_PORT')
 Rails::Command::ServerCommand.const_set('DEFAULT_PORT', 4224)
+
+rmt_path = File.expand_path('../', __dir__)
+FastGettext.add_text_domain('rmt', path: File.join(rmt_path, 'locale'), type: :po, ignore_fuzzy: true, report_warning: false)
+
+FastGettext.text_domain = 'rmt'
+FastGettext.available_locales = %w[en de fr ja es pt_BR zh_CN]
```
[the remark this answers is not in the document]
Comment (tmuntaner) [reply config/boot.rb]: This is optional, but we limit the scope of the translations to only the ones that we have available. As this is logic from before, I would like to leave it as is.

**Comment by tmuntaner** [comment]
@felixsch This pr should be open to review again.

Also, RMT translates the API as well. This means that if we want to respect translated error messages in the connect api, we have to bundle the translations in the base package. This means that we don't need the package `rmt-server-lang`.

**Pull request merged**